### PR TITLE
Go all the way in with docker-compose :)

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -99,26 +100,29 @@ The conda organization adheres to the [NumFOCUS Code of Conduct](https://www.num
...
+   By default, it will pick the Python 3.9 image built from `master`.
+
+   If you need a different Python version, set a `CONDA_DOCKER_PYTHON`
+   environment variable like this to rebuild the image:
 
    **Bash (macOS, Linux, Windows)**
 
    ```bash
-   $ CONDA_DOCKER_PYTHON=3.8 docker compose run unit-tests
+   $ CONDA_DOCKER_PYTHON=3.8 docker compose build unit-tests
```
Comment (jezdez) [inline CONTRIBUTING.md:114]: This is needed since just running the container with the env var won't trigger the rebuild.

File: Dockerfile [hunk 2]
```diff
@@ -16,18 +16,18 @@ FROM debian:buster-slim
...
 ENV PATH /opt/conda/bin:$PATH
 
-COPY ci/scripts/linux/setup.sh /tmp
+COPY dev/linux/setup.sh /tmp
 RUN bash /tmp/setup.sh
 
 COPY --from=buildbase /opt/conda /opt/conda
 
-ARG PYTHON_VERSION=3.8
+ARG python_version=3.9
```
Comment (jezdez) [inline Dockerfile:24]: Just a convention thing in Docker, args are lower case there.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jaimergp) [reply CONTRIBUTING.md:114]: Will `build` also _run_ the service command after?

Comment (jaimergp) [reply CONTRIBUTING.md:114]: I'll merge and we can continue the discussion in the main PR.

**Pull request merged**